pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -394,6 +394,10 @@ def _detect_replacable_format_call(self, node: nodes.Const) -> None:
             if "\\" in node.parent.right.as_string():
                 return
 
+            # If % applied to another type than str, it's modulo and can't be replaced by formatting
+            if not isinstance(node.parent.left.value, str):
```

Comment:
It's not safe to assume node.parent.left has `value`, it could be a tuple.

Reply:
Ok, I check now first if node.parent.left has a value attribute.